configs/wifi-idw01m1-mbedignore

```diff
@@ -0,0 +1,5 @@
+easy-connect/atmel-rf-driver/*
```

Comment:
at least esp8266 / k64f seems to compile fine without this. is there some known combination that needs this?